colosseum/wrappers.py

```diff
@@ -0,0 +1,39 @@
+from .constants import INHERIT
+
+
+class BorderSpacing:
+    """Border spacing wrapper."""
+
+    def __init__(self, values):
+        if values == INHERIT:
+            self._data = (values, )
```

Comment:
`_data` is a little generic as a name - especially since it's used consistently as `data[0]` and `data[1]`. Why not store `horizontal` and `vertical`, and do a once off determination if there is a vertical value at time of construction?

Reply:
I had that originally and thought this was simpler 😂😂

Reply:
Fixed